### ISSUE TITLE
Throwing errors for duplicate assertions 

### ISSUE THREAD

**Issue by egekorkan** [thingweb/thingweb-playground#316]
If there is an assertion in manual.csv that is also tested automatically, an error should be thrown. There needs to be an exception for https://github.com/thingweb/thingweb-playground/issues/124 though.

**Comment by FadySalama** [comment]
This should never be possible to happen in the first place. The manual.csv is generated as all assertions that are not automatically tested

**Comment by egekorkan** [comment]
Yes in theory. Just that people might have old manual.csv files that are not in sync with ours
